specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.3-preview/keys.json

```diff
@@ -1323,7 +1323,9 @@
             "RSA",
             "RSA-HSM",
             "oct",
-            "oct-HSM"
+            "oct-HSM",
+            "OKP",
+            "OKP-HSM"
```

Comment:
This is news to the Azure SDK team for 7.3-preview. Should this be in 7.4-preview?

/cc @jlichwa @maorleger

Reply:
If it is intended for 7.3-preview, is it supported in Key Vault and or Managed HSM? Do RPs in the public clouds already support it? If not, when? We need time to add support, write up tests, and record new sessions.

Reply:
Yes I was not aware of this either, and I agree that it is likely 7.4....

Reply:
Discussed this offline with Jack. I will cancel this PR, wait for 7.4-preview availability, and resubmit.

Reply:
Regarding 7.4-preview, what is the ETA for the branch/folder?

Reply:
It's already created but needs a PR to fully prep it. @jhendrixMSFT or @lmazuel  can you merge PR #15900 ?